### ISSUE TITLE
USEM: stage source MODS xml files

### ISSUE THREAD

**Issue by tcatapano** [cul/mods_for_hyacinth_import#1]
@elohanlon : Is this the complete set?:

https://github.com/cul/mods_for_hyacinth_import/tree/master/universityseminars_batch1

**Comment by elohanlon** [comment]
@tcatapano Yes it is.  That's the full set for everything that's currently in the restricted access university seminars site (35,845 records).
